index.js

```diff
@@ -466,33 +498,36 @@ module.exports = {
       findOrCreateAdditionsFeed((err) => {
         // prettier-ignore
         if (err) return cb(clarify(err, 'Error finding or creating additions feed when starting ssb-tribes2'))
-        return cb()
+        return cb(null)
       })
 
       ssb.metafeeds.findOrCreate((err, myRoot) => {
         // prettier-ignore
-        if (err) return cb(clarify(err, 'Error getting own root in start()'))
+        if (err) return console.log(clarify(err, 'Error getting own root in start()'))
```

Comment:
in `start()` you had multiple places you were calling cb.
I changed it so that cb is only called once and subsequent errors are just logged.

Reply:
mm smart. but do we want something more self-destructive than a console.log? (also afraid the logs will be caught in the linter lol)

Reply:
I don't know, but `cb` was not an option!